src/pointsdk/index.d.ts

```diff
@@ -101,6 +101,14 @@ export type ZProxyWS = WebSocket & {
     ) => Promise<() => Promise<T>>;
 };
 
+export type IdentityData = {
+    identityRegistred: boolean;
```

Comment:
There's a typo here but I didn't want to change it because that's the exact name that the API (in Point Engine) returns, and changing the API payload could cause issues in other places.